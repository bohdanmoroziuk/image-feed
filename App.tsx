import { View } from 'react-native';
import { StatusBar } from 'expo-status-bar';

import styles from './App.styles';

import Avatar from 'src/components/Avatar';

export default function App() {
  return (
    <View style={styles.app}>
      <StatusBar style="auto" />
      <Avatar color="teal" initials="FL" size={35} />
    </View>
  );
}
