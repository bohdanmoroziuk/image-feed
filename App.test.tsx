import { render } from '@testing-library/react-native';

import App from './App';

describe('App', () => {
  it('renders successfully', () => {
    const { toJSON } = render(<App />);

    expect(toJSON()).toMatchSnapshot();
  });
});
