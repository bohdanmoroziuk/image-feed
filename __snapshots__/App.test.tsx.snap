// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders successfully 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "marginTop": 20,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        },
        Object {
          "backgroundColor": "teal",
          "borderRadius": 17.5,
          "height": 35,
          "width": 35,
        },
      ]
    }
  >
    <Text
      style={
        Object {
          "color": "white",
        }
      }
    >
      FL
    </Text>
  </View>
</View>
`;
